tap_counter/ViewController.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  ViewController.swift
+//  tap_counter
+//
+//  Created by Michail on 9/10/16.
+//  Copyright © 2016 mishaparadising. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    
+    var counter = 0;
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view, typically from a nib.
+        
+        tapCounter.text = String(counter);
+        
+        self.navigationItem.hidesBackButton = true;
+        
+        let resetButton = UIBarButtonItem(title: "Reset", style: UIBarButtonItemStyle.Bordered, target: self, action: Selector("resetButton"))
```

Comment:
So this is an ongoing discussion, you can check it out [here](https://github.com/JustForFunOrg/12-Add-New-Item/pull/1#issuecomment-242896045). In short, as we discussed, I prefer to separate long definitions on line-by-line basis. 

For example, this can be rewritten as 
```
        let resetButton = UIBarButtonItem(
            title: "Reset",
            style: UIBarButtonItemStyle.Plain,
            target: self,
            action: #selector(resetButtonTapped))
```